Add contributing section to README (#1242)
googleapis/api-linter: README.md

This is an attempt to follow the spirit of semantic versioning while still
being useful.

## Contributing

> We are thrilled that you are interested in contributing to the API linter. Please look into the [guide](./docs/contributing.md) to learn more.

Reviewer: ```suggestion
If you are interested in contributing to the API linter, please review the [contributing guide](https://linter.aip.dev/contributing) to learn more.
```
